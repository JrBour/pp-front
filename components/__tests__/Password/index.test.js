import { shallowMount } from '@vue/test-utils'
import Password from '@/components/Password'

describe('Password', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Password, {
      propsData: {
        rules: "required",
        onChange: jest.fn(),
        name: 'password',
      },
    })
  })

  describe('snapshot', () => {
    it('renders properly', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
