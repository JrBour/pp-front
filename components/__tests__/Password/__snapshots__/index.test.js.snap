// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Password snapshot renders properly 1`] = `
<div class="input__wrapper">
  <validationprovider-stub vid="" mode="aggressive" rules="required" bails="true" skipifempty="true" debounce="0" tag="div" custommessages="[object Object]" detectinput="true" class="input__container"></validationprovider-stub>
</div>
`;
